packages/nhsuk-frontend/src/nhsuk/template.njk

```diff
@@ -2,28 +2,32 @@
 {% from "nhsuk/components/skip-link/macro.njk" import skipLink -%}
 {% from "nhsuk/components/header/macro.njk" import header -%}
 {% from "nhsuk/components/footer/macro.njk" import footer -%}
+
+{% set themeColor = themeColor | default("#005eb8") -%}
```

Comment:
Thanks for removing all the `assetPath` defaults

We missed the fallback at the top though!

```suggestion
{% set themeColor = themeColor | default("#005eb8") -%}
{% set assetPath = assetPath | default("/assets", true) -%}
```